example/CMakeLists.txt

```diff
@@ -22,7 +22,8 @@
 
 cmake_minimum_required(VERSION 3.18)
 
-project("alpakaExamples")
+project("alpakaExamples"
+        LANGUAGES CXX)
```

Comment:
Any specific reason for the newline here? Why not:
```suggestion
project("alpakaExamples" LANGUAGES CXX)
```

Reply:
No, this has no deeper meaning. I can change it if you want.

Reply:
I prefer the single line, but I don't want to burn the CI time for it. Do as you wish.

Reply:
Fixed. This is also more consistent with the top-level `CMakeLists.txt`.